Sources/PrimerSDK/Classes/User Interface/Text Fields/PrimerGenericTextFieldView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  PrimerGenericTextFieldView.swift
+//  PrimerSDK
+//
+//  Created by Evangelos on 12/11/21.
+//
+
+#if canImport(UIKit)
+
+import UIKit
+
+public final class PrimerGenericFieldView: PrimerTextFieldView {
+    
+    public var allowedCharacterSet: CharacterSet?
+    public var maxCharactersAllowed: UInt?
+    public var shouldMaskText: Bool = false
+    public override var text: String? {
+        get {
+            return shouldMaskText ? "****" : textField._text
```

Comment:
Apologise for the dumb 🤪 question - was just checking whether this `text` is just for display purposes. Thanks!